### PR TITLE
Removing CSharp validator 

### PR DESCRIPTION
**Pull request by dsgouda**
cheers from @olydis, acknowledged your comment but python is still failing, removing any text that seems special here.

### CODE REVIEW

**Comment by olydis** [comment]
ah, actually you can move https://github.com/Azure/autorest/tree/master/docs/developer/validation-rules too 🙂 ...and then just link to the new location of the docs in here

File: src/autorest-core/resources/default-configuration.md [hunk 1]
```diff
@@ -16,7 +16,9 @@ openapi-type: arm
 
 ``` yaml
 use-extension:
-  "@microsoft.azure/autorest-classic-generators": 2.0.0-preview
+  "@microsoft.azure/autorest-classic-generators": "2.0.0-preview"
+  "@microsoft.azure/classic-openapi-validator": "C:\\repos\\azure-openapi-validator\\src\\dotnet\\AutoRest\\microsoft.azure-classic-openapi-validator-1.0.0.tgz"
```
Comment (olydis) [inline src/autorest-core/resources/default-configuration.md]: pending

File: src/autorest-core/resources/default-configuration.md [hunk 2]
```diff
@@ -105,23 +107,22 @@ pipeline:
 
 ##### Azure Validator
```
Comment (olydis) [inline src/autorest-core/resources/default-configuration.md]: why is this still here???

Comment (olydis) [inline src/autorest-core/resources/default-configuration.md]: @dsgouda just raising awareness, this is still here

Comment (dsgouda) [inline src/autorest-core/resources/default-configuration.md]: I swear I removed this!

**Comment by dsgouda** [comment]
Assuming Travis passes.

**Comment by olydis** [comment]
Travis seems to be super overbooked today... stuff not even starting to execute forever

**Comment by dsgouda** [comment]
Republished a new preview version of the classic openapi validator without additional deps. 
Testing to check if this works.

**Pull request merged**